### PR TITLE
[Improve][Connector-V2] Time supports default value

### PR DESCRIPTION
**Pull request by corgy-w**
### Purpose of this pull request

Time supports default value and keyword

eg：
```
{
              name = book_publication_time
              type = timestamp
              defaultValue = "2024-09-12 15:45:30"
              comment = "book publication time"
           },
           {
              name = book_publication_time2
              type = timestamp
              defaultValue = CURRENT_TIMESTAMP
              comment = "book publication time2"
           },
           {
              name = book_publication_time3
              type = time
              defaultValue = "15:45:30"
              comment = "book publication time3"
           },
           {
              name = book_publication_time4
              type = time
              defaultValue = CURRENT_TIME
              comment = "book publication time4"
           },
           {
              name = book_publication_time5
              type = date
              defaultValue = "2024-09-12"
              comment = "book publication time5"
           },
           {
              name = book_publication_time6
              type = date
              defaultValue = CURRENT_DATE
              comment = "book publication time6"
           }

or 

schema = {
      fields {
        c_map = "map<string, string>"
        c_double = double
        c_timestamp = timestamp
      }
    }
    rows = [
       {
         kind = INSERT
         fields = [{"__name__": "test1"},  1.23, CURRENT_TIMESTAMP]
     }]
```

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java [hunk 1]
```diff
@@ -70,7 +70,9 @@ public class JsonToRowConverters implements Serializable {
                     .toFormatter();
 
     public static final String FORMAT = "Common";
-
+    private static final String CURRENT_DATE = "CURRENT_DATE";
+    private static final String CURRENT_TIME = "CURRENT_TIME";
+    private static final String CURRENT_TIMESTAMP = "CURRENT_TIMESTAMP";
```
Comment (Hisoka-X) [inline seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java]: It's weird to put this into JsonToRowConverters. Why not just put it into fake source connector?

Comment (corgy-w) [reply seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java]: I also discovered it and changing it.

File: seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java [hunk 2]
```diff
@@ -264,7 +263,8 @@ private LocalDate convertToLocalDate(JsonNode jsonNode, String fieldName) {
     }
 
     private LocalTime convertToLocalTime(JsonNode jsonNode) {
-        TemporalAccessor parsedTime = TIME_FORMAT.parse(jsonNode.asText());
+        String localTime = jsonNode.asText();
+        TemporalAccessor parsedTime = TIME_FORMAT.parse(localTime);
```
Comment (Hisoka-X) [inline seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java]: please revert this change.

Comment (corgy-w) [reply seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java]: get

**Review by Hisoka-X — DISMISSED** [review]
Please update the doc

File: docs/en/connector-v2/source/FakeSource.md [hunk 3]
```diff
@@ -401,15 +401,86 @@ source {
 }
 ```
 
+### Complete `schema` Case
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/FakeSource.md]:
```suggestion
### Options `defaultValue` Case
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @corgy-w

**Review by hailin0 — APPROVED** [review]
